src/components/shared-tabs.scss

```
@@ -0,0 +1,128 @@
+    opacity: 1;
+    border: var(
+      --home-features-item-bg-border,
+      1px solid rgba(17, 17, 19, 0.08)
+    );
+  }
+}
+
+.shared-tabs__card-icon {
+  width: 24px;
+  height: 24px;
+  background-color: currentColor;
```

Stylelint (17.11.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)